src/pages/account-menu/in-an-app-bar/Chip.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Chip as MuiChip, ChipProps as MuiChipProps, Typography, makeStyles, Theme, useTheme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    userMenuChip: {
+        width: '112px',
```

Comment:
Is there a way to do this without a fixed width? Presumably the label can change and won't always look correct with a fixed width.